doc/architecture.md

```diff
@@ -43,7 +43,7 @@ The build image manifest is at
 [/images/build/Dockerfile](../images/build/Dockerfile). It is built and
 published via the Microsoft Container Registry (MCR) ([info][]) as
 `mcr.microsoft.com/oryx/build` and syndicated to Docker Hub as
-[https://hub.docker.com/_/microsoft-oryx-images][]. Pull with `docker pull
+[`https://hub.docker.com/_/microsoft-oryx-images`](https://hub.docker.com/_/microsoft-oryx-images). Pull with `docker pull
```

Comment:
We've turned off syndication. @arroyc can you please confirm?

Reply:
syndication is off for individual images but it's there for oryx family of images